Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM opensuse/leap:15.4
+RUN zypper install -y --no-recommends git dbus-1 systemd-sysvinit
```

Comment:
Maybe we need to add `# hadolint ignore=DL3037` here, but I think the `zypper clean` should be used.

Reply:
`clean` is a good idea, but I don' think we should pin the versions installed.
If a newer version breaks anything I'd like to know.

Reply:
> I don' think we should pin the versions installed

I don't think this too. That's why I suggested to include `# hadolint ignore=DL3037`, to silence this check.

Reply:
Apparently the command didn't work.